### DISCUSSION TITLE
Reviewers + AutoMerges

### DISCUSSION THREAD

**Discussion by KnisterPeter** [renovatebot/renovate#14407] [Suggest an Idea]
Hi there :wave:,

I have a question about reviewers and automerges within renovate.
I've search through the docs and found this: https://docs.renovatebot.com/key-concepts/automerge/#assignees-and-reviewers

I was wondering in our case, why some PRs don't get reviewers assigned at all. We have repository which requires approvals by at least one reviewer (GitHub) and we have bot which does make an approval if certain conditions are met.
But now, since renovate doesn't request a review, the bot isn't triggered to do a review.
Would it be possible to add an option to renovate to add reviewers even though automerges are enabled?

**Comment by viceice** [comment]
Nope, renovate won't assign reviewers or assignees on automerge to reduce noise.

**Reply by KnisterPeter** [reply]
Yes, I've seen this and references it here from the docs as well.
Since I've described our use-case, would it be a valid for a feature request?

**Reply by rarkins** [reply]
Already supported, if I understand you right: [assignAutomerge](https://docs.renovatebot.com/configuration-options/#assignautomerge)

**Reply by KnisterPeter** [reply]
Awesome thank you. Didn't know that
